### DISCUSSION TITLE
Custom Tags

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#162] [General]
In the next version:

![customtags](https://github.com/user-attachments/assets/88ef3d69-55c9-4490-aecd-4eb59f3e5199)

In the `Tags` category you can define custom tags.

These generate commands you can use.

Like `Tag: later`

You can use this to apply tags to tabs quickly in various ways.

**Comment by N3C2L** [comment]
Great idea! This is a even faster way than the "auto tag picker". Thanks!
